homeassistant/components/config/config_entries.py

```diff
@@ -678,6 +678,7 @@ async def config_subentry_list(
     result = [
         {
             "subentry_id": subentry.subentry_id,
+            "subentry_type": subentry.subentry_type,
```

Comment:
Do we need to wait for a frontend PR for this change?

Reply:
No, the draft frontend PR already supports this.

Reply:
I'd link that frontend PR in this PR description.